packages/schematics/src/ng-add/files/main.ts

```diff
@@ -10,9 +10,9 @@ function createWindow() {
   mainWindow = new BrowserWindow({ width: 800, height: 600 });
 
   // and load the index.html of the app.
-  const isServe = process.argv.includes('--serve');
+  const isServe = process.env.ELECTRON_SERVE === 'true';
```

Comment:
You could change that to:
```typescript
const isServe = !!process.env.ELECTRON_SERVE;
```

Reply:
`ELECTRON_SERVE` is of type string, so even if is value is 'false' `!!ELECTRON_SERVE` will be true.